test/_utils_internal.py

```diff
@@ -40,6 +42,13 @@ def get_available_devices():
             devices += [torch.device(f"cuda:{i}")]
             if i == 1:
                 break
+    if is_npu_available():
+        n_npu = torch.npu.device_count()
+        if n_npu > 0:
+            for i in range(n_npu):
+                devices += [torch.device(f"npu:{i}")]
+                if i == 1:
+                    break
```

Comment:
I'm open to this but I would put big warning as a comment and a real warning as well to tell users that this is an experimental and untested (as in: not part of the CI) feature. Can you do that?